src/selectors/getCursorSortDirection.ts

```diff
@@ -0,0 +1,15 @@
+/** A selector that returns the sort direction of the cursor context. */
+import { State } from '../util/initialState'
+import { getSortPreference, simplifyPath } from './index'
+import { HOME_PATH } from '../constants'
+import { pathToContext } from '../util'
+
+/** Get sort direction of cursor. */
+const getCursorSortDirection = (state: State) => {
```

Comment:
It would be better to define this inline in the `toggleSort` module since it's not used anywhere else. This helps reduce the surface area of the internal API.